code/crates/starknet/host/src/node.rs

```diff
@@ -251,7 +251,7 @@ fn make_config(index: usize, total: usize, settings: MakeConfigSettings) -> Conf
         moniker: format!("starknet-{index}"),
         consensus: ConsensusConfig {
             value_payload: ValuePayload::PartsOnly,
-            queue_capacity: 100,
+            queue_capacity: 100, // Deprecated, derived from `sync.parallel_requests`
```

Comment:
Can't we just remove it for our test apps?

Reply:
I see. But when we say "This setting is deprecated and will be removed in the future." what is the final solution? The 1..3 steps you describe above?

Reply:
#1150